croncheck.php

```diff
@@ -31,6 +31,15 @@
  */
 
 define('NO_UPGRADE_CHECK', true);
+// @codingStandardsIgnoreStart
+if (isset($argv)) {
+    define('CLI_SCRIPT', true);
+} else {
+    define('CLI_SCRIPT', false);
+}
+//@codingStandardsIgnoreEnd
+require('../../../config.php');
+require_once('iplock.php');
```

Comment:
I thnk this code diff chunk isn't needed, we only need to iplock function if we actually check for ip's and we don't need to do that at all if we run this from the cli.

So I think the require_once and then the check itself can be moved to line 103 to just after where the config.php was originally